diff -Nur hugo-2.10.orig/debug.c hugo-2.10.chng/debug.c
--- hugo-2.10.orig/debug.c	Wed Feb 19 23:56:32 2003
+++ hugo-2.10.chng/debug.c	Thu Sep 25 10:16:46 2003
@@ -48,8 +48,8 @@
 {
   register char __AUX;
 
-  __asm__ __volatile__ ("movl _Page (, %%eax, 4), %%ecx
-			movb (%%edx, %%ecx), %%al "
+  __asm__ __volatile__ ("movl _Page (, %%eax, 4), %%ecx "
+			"movb (%%edx, %%ecx), %%al "
                         :"=a" (__AUX)
                         :"d" (A),
 			"a" (A >> 13):"%ebx", "%ecx");
